core/src/main/resources/jenkins/split-plugins.txt

```diff
@@ -30,4 +30,4 @@ jdk-tool 2.112 1.0
 jaxb 2.163 2.3.0 11
 
 #JENKINS-43610 Split Trilead out from Core
-trilead-api 2.181 1.0.3 8
+trilead-api 2.182 1.0.4
```

Comment:
For now, to make this build, this would have to be an incremental version, as I did for example in #3988.